dill/session.py

```diff
@@ -7,19 +7,42 @@
 # License: 3-clause BSD.  The full license text is available at:
 #  - https://github.com/uqfoundation/dill/blob/master/LICENSE
 """
-Pickle and restore the intepreter session.
+Pickle and restore the intepreter session or a module's state.
+
+The functions :py:func:`dump_module`, :py:func:`load_module` and
```

Comment:
The doc here is good, but I think could use some rework with regard to the English. I'll come back to this.